init.lua

```diff
@@ -144,11 +147,19 @@ local function register_air_tank(name, desc, color, uses, material)
 		inventory_image = "airtanks_airtank.png^[colorize:"..color.."^[mask:airtanks_airtank.png",
 		wield_image = "airtanks_airtank.png^[colorize:"..color.."^[mask:airtanks_airtank.png",
 		stack_max = 1,
+		_mcl_armor_element = "torso",
+		_mcl_armor_texture = "airtanks_chestplate_tank.png",
 	
 		on_place = function(itemstack, user, pointed_thing)
-			return use_airtank(itemstack, user)
+		   if minetest.get_modpath("mcl_armor") then
+		      mcl_armor.equip_on_use(itemstack, user, pointed_thing)
```

Comment:
Is there no return value to forward to `on_place`?

EDIT: Also for L208+ and any other occurrences

Reply:
fixed